Content.Server/Gravity/EntitySystems/WeightlessSystem.cs

```diff
@@ -112,10 +108,19 @@ private void EntParentChanged(ref EntParentChangedMessage ev)
                 }
             }
 
-            var newGrid = EntityManager.GetComponent<TransformComponent>(ev.Entity).GridID;
+            var newGrid = ev.Transform.GridID;
             var newStatuses = _alerts.GetOrNew(newGrid);
 
             newStatuses.Add(status);
+
+            // then update the actual alert.
+            // only showing the alert is the user is on a grid that is explicitly missing gravity.
+            if (!_mapManager.TryGetGrid(newGrid, out var grid)
+                || !TryComp(grid.GridEntityId, out GravityComponent? gravity)
+                || gravity.Enabled)
+                RemoveWeightless(status.Owner);
+            else
+                AddWeightless(status.Owner);
```

Comment:
Or should the alert also show up in space?

Reply:
I'm leaning towards having weightless show in space.

Reply:
Well if planets aren't just big grids then I guess this'd need to be updated whenever planets and the like get added.

Reply:
It now shows up in space, and also checks for `MovementIgnoreGravityComponent`.

AFAIK there is currently no way for an entity to get or loose that component. If there is, then the alert also needs updating whenever that gets added/removed.